state-chain/pallets/cf-funding/src/lib.rs

```diff
@@ -973,3 +1063,67 @@ impl<T: Config> OnKilledAccount<T::AccountId> for Pallet<T> {
 		BoundRedeemAddress::<T>::remove(account_id);
 	}
 }
+
+#[derive(Clone, PartialEq, Eq, Encode, Decode, TypeInfo, DebugNoBound)]
+#[scale_info(skip_type_params(T))]
+pub enum AllowedCallsViaSCGateway<T: Config> {
```

Comment:
Somewhat pedantic point, but the calls are not via the sc gateway. They are via SC utils. 

Let's just call it the `EthereumInterface`, or something.